packages/core/content-releases/admin/src/pages/ReleaseDetailsPage.tsx

```diff
@@ -21,69 +35,363 @@ import styled from 'styled-components';
+                defaultMessage: '{number, plural, =0 {No entries} one {# entry} other {# entries}}',
+              },
+              { number: totalEntries }
+            )
+          }
+          navigationAction={
+            <Link startIcon={<ArrowLeft />} to="/plugins/content-releases">
+              {formatMessage({
+                id: 'global.back',
+                defaultMessage: 'Back',
+              })}
+            </Link>
+          }
+          primaryAction={
+            <Flex gap={2}>
+              <IconButton
+                label={formatMessage({
+                  id: 'content-releases.header.actions.open-release-actions',
+                  defaultMessage: 'Release actions',
+                })}
+                ref={moreButtonRef}
+                onClick={handleTogglePopover}
+              >
+                <More />
+              </IconButton>
+              {isPopoverVisible && (
+                <Popover
+                  source={moreButtonRef}
+                  placement="bottom-end"
+                  onDismiss={handleTogglePopover}
+                  spacing={4}
+                  minWidth="242px"
+                >
+                  <Flex alignItems="center" justifyContent="center" direction="column" padding={1}>
+                    <CheckPermissions permissions={PERMISSIONS.update}>
+                      <PopoverWrapper onClick={openReleaseModal}>
+                        <PencilIcon />
+                        <Typography ellipsis>
+                          {formatMessage({
+                            id: 'content-releases.header.actions.edit',
+                            defaultMessage: 'Edit',
+                          })}
+                        </Typography>
+                      </PopoverWrapper>
+                    </CheckPermissions>
+                    <PopoverWrapper>
+                      <TrashIcon />
+                      <Typography ellipsis textColor="danger600">
+                        {formatMessage({
+                          id: 'content-releases.header.actions.delete',
+                          defaultMessage: 'Delete',
+                        })}
+                      </Typography>
+                    </PopoverWrapper>
+                  </Flex>
+                  <ReleaseInfoWrapper
+                    direction="column"
+                    justifyContent="center"
+                    alignItems="flex-start"
+                    gap={1}
+                    padding={5}
+                  >
+                    <Typography variant="pi" fontWeight="bold">
+                      {formatMessage({
+                        id: 'content-releases.header.actions.created',
+                        defaultMessage: 'Created',
+                      })}
+                    </Typography>
+                    <Typography variant="pi" color="neutral300">
+                      <RelativeTime timestamp={new Date(release?.createdAt)} />
```

Comment:
I believe you have a ts error here because release might be undefined. See my comment above to safely access the values in the success return